static/src/javascripts/projects/common/modules/component.js

```diff
@@ -107,7 +109,11 @@ define([
 
         this.elem = template;
         this._prerender();
-        bonzo(container)[this.manipulationType](this.elem);
+
+        fastdom.write(function () {
+            bonzo(container)[this.manipulationType](this.elem);
+        });
+
         this._ready();
```

Comment:
To whoever is checking this PR. Does this `ready()` call need to move into the `fastdom(...)` call? Feels like you want to minimise `fastdom(...)` to only things that manipulate the dom. However it feels like this might cause race conditions.